app/scripts/controllers/about.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+
+angular.module('openhimConsoleApp')
+  .controller('AboutCtrl', function ($rootScope, $scope, Api, Alerting, Notify, config) {
+    
+    var success = function(result) {
+      $scope.aboutInfo = result;
+      Notify.notify('aboutInfoReceived');
+    };
+    
+    var error = function(err) {
+      Alerting.AlertAddServerMsg(err.status);
+    };
+    
+    Api.About.query(success, error);
+    
+    
+    var getVersionCompatible = function(minVersion, actualVersion) {
```

Comment:
maybe a more description name for the function would be 'isCoreVersionCompatible'